join_github_app/main/routes/join.py

```diff
@@ -79,7 +79,21 @@ def join_selection():
 
 @join_route.route("/invitation-sent")
 def invitation_sent():
-    return render_template("pages/invitation-sent.html")
+    email = session.get("email", "").lower()
```

Comment:
Worth using the email provided by Auth0 instead of the user-provided email - just in case there are any issues, it'll show them the email they've authenticated against rather than the email they provided 😁 

I believe the email provided by Auth0 is the one we actually send the error to (and if it's not, it should be 🔐)

I believe we also check whether the emails are the same, so a bit of a non-issue, but is an extra layer of cheese for security 🧀